ci/konflux_minikube_e2e_tests.sh

```
@@ -0,0 +1,128 @@
+chmod 600 minikube-ssh-ident
+
+ssh -o StrictHostKeyChecking=no $MINIKUBE_USER@$MINIKUBE_HOST -i minikube-ssh-ident "minikube delete"
+ssh -o StrictHostKeyChecking=no $MINIKUBE_USER@$MINIKUBE_HOST -i minikube-ssh-ident "minikube start --cpus 6 --disk-size 10GB --memory 16000MB --kubernetes-version=1.26.3 --addons=metrics-server --disable-optimizations"
+
+export MINIKUBE_IP=`ssh -o StrictHostKeyChecking=no $MINIKUBE_USER@$MINIKUBE_HOST -i minikube-ssh-ident "minikube ip"`
+
+set -x
+
+scp -o StrictHostKeyChecking=no -i minikube-ssh-ident $MINIKUBE_USER@$MINIKUBE_HOST:$MINIKUBE_ROOTDIR/.minikube/profiles/minikube/client.key ./
+scp -i minikube-ssh-ident $MINIKUBE_USER@$MINIKUBE_HOST:$MINIKUBE_ROOTDIR/.minikube/profiles/minikube/client.crt ./
+scp -i minikube-ssh-ident $MINIKUBE_USER@$MINIKUBE_HOST:$MINIKUBE_ROOTDIR/.minikube/ca.crt ./
+
+ssh -o ExitOnForwardFailure=yes -f -N -L 127.0.0.1:8444:$MINIKUBE_IP:8443 -i minikube-ssh-ident $MINIKUBE_USER@$MINIKUBE_HOST
+
+cat > kube-config <<- EOM
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: $PWD/ca.crt
+    server: https://127.0.0.1:8444
+  name: 127-0-0-1:8444
+contexts:
+- context:
+    cluster: 127-0-0-1:8444
+    user: remote-minikube
+  name: remote-minikube
+users:
+- name: remote-minikube
+  user:
+    client-certificate: $PWD/client.crt
+    client-key: $PWD/client.key
+current-context: remote-minikube
+kind: Config
+preferences: {}
+EOM
+
+export PATH="$KUBEBUILDER_ASSETS:$PATH"
+export PATH="/root/go/bin:$PATH"
+
+export KUBECONFIG=$PWD/kube-config
+export KUBECTL_CMD="kubectl "
+$KUBECTL_CMD config use-context remote-minikube
+$KUBECTL_CMD get pods --all-namespaces=true
+
+source build/kube_setup.sh
+
+export IMAGE_TAG=`git rev-parse --short=8 HEAD`
+
+$KUBECTL_CMD create namespace clowder-system
+
+mkdir artifacts
+
+make release
+
+cat manifest.yaml > artifacts/manifest.yaml
+
+sed -i "s/clowder:latest/clowder:on-pr-$IMAGE_TAG/g" manifest.yaml
```

This shouldn't work? because `on-pr` is removed. Can we investigate how this is working? Maybe this line is just silently doing nothing in the script now? We need to figure this out.